packages/autorest.python/autorest/codegen/models/property.py

```diff
@@ -130,31 +130,46 @@ def validation(self) -> Optional[Dict[str, Any]]:
         return retval or None
 
     @staticmethod
-    def contain_model_type(t: BaseType) -> bool:
-        from . import ListType, DictionaryType, ModelType
+    def contain_target(t: BaseType, check_target: Callable[[BaseType], bool]) -> bool:
```

Comment:
what's the diff between model_type and target?

Reply:
Hi @iscai-msft , the origin `contain_model_type` is to check whether the input contains `ModelType`. And now I want to reuse logic in this function to check exists of `named CombinedType`, so to change this function to find by an callback criteria.
The usage of this function is at https://github.com/Azure/autorest.python/blob/20230202-namedunion/packages/autorest.python/autorest/codegen/models/property.py#L156-L167